recipes/esys-escript/meta.yaml

```diff
@@ -0,0 +1,75 @@
+
+package:
+    name: esys-escript
+    version: 5.4
+source:
+    - svn_url: https://svn.geocomp.uq.edu.au/svn/esys13/trunk
```

Comment:
See https://conda-forge.org/docs/maintainer/adding_pkgs.html#tarballs-no-repos

Reply:
There is a tarball of escript that I can use if that is preferred. I have changed the meta.yaml file to add this in.

I have also pinned the version of scons, as escript it will fail to compile for versions older than 3.0.5 and for version of scons that do not use python 2.7.